Themes/default/languages/Help.english.php

```diff
@@ -616,7 +616,7 @@
 $helptxt['alert_pm_new'] = 'Notifications of new personal messages do not appear in the Alerts pane, but appear in the "My Messages" list instead.';
 $helptxt['alert_event_new'] = 'This will send out an alert or email as requested if there is a new calendar event added. However, if that event is posted and a topic is added, you will not get an alert for the event if you\'re already following that board - the alert from following the board would cover this.';
 
-$helptxt['force_ssl'] = '<strong>Test SSL and HTTPS on your server properly before enabling this, it may cause your forum to become inaccessible.</strong> Enable maintenance mode if you are unable to access the forum after enabling this';
+$helptxt['force_ssl'] = '<strong>Test SSL and HTTPS on your server properly before enabling this, it may cause your forum to become inaccessible.</strong> Enable maintenance mode if you are unable to access the forum after enabling this.<br><br><strong>Changing this setting will update all forum URLs to match your selection.</strong>';
```

Comment:
The phrase "all forum URLs" might confuse people. Naive admins might think this will update URLs in the text of forum posts, while knowledgable admins will be worried that customized URLs in their settings will be overwritten. I suggest something like this:

`Changing this setting will update "' . $boardurl . '" to "' . (str_replace('http:', 'https:', $boardurl) . '" in several key settings for your forum, including the URLs for your theme files and images, smileys, avatars, etc. Customized URLs in these settings will not be affected.`

Reply:
Done.  Let me know if you like the new wording.  I was hesitant to use $boardurl in the message, because that there are times when it doesn't match Settings.php.   (Not only .htaccess redirects, but SMF also modifies http headers with a 302 temp redirect as well when force_ssl =2...)  It was confusing to me to see a value displayed that I knew was wrong.